docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -362,6 +362,16 @@ TestCafe actions can interact with elements if they satisfy the following condit
 
 ## Examples
 
+### Disable the 'Close Multiple Tabs' Popup in IE
+
+Internet Explorer displays a native dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and halts tests. Disable this popup before you run tests.
+
+To disable the popup, manually open multiple tabs in one window of Internet Explorer. In the dialog window that appears, check `Always close all tabs` and click `Close all tabs`. IE saves this setting and the popup does not appear again.
```

Comment:
> To disable the popup, manually open multiple tabs in one window of Internet Explorer. In the dialog window that appears

One step missing. The dialog window won't appear after you open multiple tabs. You need to close the window to show it.

Reply:
fixed